docs/detections/prebuilt-rules/rule-details/bash-shell-profile-modification.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[bash-shell-profile-modification]]
+=== Bash Shell Profile Modification
+
+Both ~/.bash_profile and ~/.bashrc are files containing shell commands that are run when Bash is invoked. These files are executed in a user's context, either interactively or non-interactively, when a user logs in so that their environment is set correctly. Adversaries may abuse this to establish persistence by executing malicious content triggered by a user’s shell.
+
+*Rule type*: query
+
+*Rule indices*:
+
+* logs-endpoint.events.*
+* auditbeat-*
+
+*Severity*: medium
+
+*Risk score*: 47
+
+*Runs every*: 5 minutes
+
+*Searches indices from*: now-9m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*:
+
+* https://www.anomali.com/blog/pulling-linux-rabbit-rabbot-malware-out-of-a-hat
+
+*Tags*:
+
+* Elastic
+* Host
+* macOS
+* Linux
+* Threat Detection
+* Persistence
+
+*Version*: 1
+
+*Added ({stack} release)*: 7.12.0
+
+*Rule authors*: Elastic
+
+*Rule license*: Elastic License v2
+
+==== Potential false positives
+
+Changes to the Shell Profile tend to be noisy, a tuning per your environment will be required.
+
+==== Rule query
+
+
+[source,js]
+----------------------------------
+event.category:file and event.type:change and process.name:(* and not
+(sudo or vim or zsh or env or nano or bash or Terminal or xpcproxy or
+login or cat or cp or launchctl or java)) and not
+process.executable:(/Applications/* or /private/var/folders/* or
+/usr/local/*) and file.path:(/private/etc/rc.local or /etc/rc.local or
+/home/*/.profile or /home/*/.profile1 or /home/*/.bash_profile or
+/home/*/.bash_profile1 or /home/*/.bashrc or /Users/*/.bash_profile or
+/Users/*/.zshenv)
+----------------------------------
```

Comment:
Looks like parts of the query are being rendered as comments in the code snippet.

<img width="783" alt="image" src="https://user-images.githubusercontent.com/56409778/112062054-fd519d80-8b24-11eb-88ff-0a207949015d.png">

Reply:
Nice find. I played with some of the other code escape blocks and understandably, this syntax causes issues with all of them. Most are interpreting the `/* ... */` as block comments, which is just KQL syntax. The only option at that point would be to do a generic escape with no syntax highlighting for these rules, but I think that is too much and creates inconsistency.

I think we are to leave as is until someone complains